src/main/java/org/prebid/server/privacy/gdpr/vendorlist/VendorListService.java

```diff
@@ -286,7 +286,9 @@ private static boolean isValid(VendorList vendorList) {
      * Handles errors occurred while HTTP request or response processing.
      */
     private static Future<Void> failResponse(Throwable exception, int version) {
-        logger.warn("Error fetching vendor list via HTTP for version {0}", exception, version);
+        logger.warn("Error fetching vendor list via HTTP for version {0} with message: {1}",
+                version, exception.getMessage());
+        logger.debug("Error fetching vendor list via HTTP {0}", version, exception);
```

Comment:
Seems you've wrong sequence of `version, exception` arguments order.

Reply:
I'll change it.